form.yml.erb

```diff
@@ -7,6 +7,7 @@
 cluster:
   - "owens"
   - "pitzer"
+  - "cardinal"
```

Comment:
Shouldn't we take this opportunity to remove owens?

Reply:
Yes, let me remove the Owens related items and then update the branch.

Reply:
I have removed Owens.